bitnami/kubernetes-event-exporter/Chart.yaml

```diff
@@ -31,4 +31,4 @@ maintainers:
 name: kubernetes-event-exporter
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/kubernetes-event-exporter
-version: 2.9.3
+version: 2.9.4
```

Comment:
This should be a minor bump, we are adding a new feature (default configurable probes)

Reply:
ok! Changed to 2.10.0 :)